hack/kind/setup.sh

```diff
@@ -1,8 +1,10 @@
 #!/usr/bin/env bash
 set -e -u -o pipefail
 
-declare -r SCRIPT_PATH=$(readlink -f "$0")
-declare -r SCRIPT_DIR=$(cd $(dirname "$SCRIPT_PATH") && pwd)
+declare -r SCRIPT_PATH
+SCRIPT_PATH=$(readlink -f "$0")
+declare -r SCRIPT_DIR
```

Comment:
This should cause the script to fail since we are modifying the variable that is declared readonly. I think it is safe to turn off this lint than assign and then mark the variable readonly. 

See: https://github.com/koalaman/shellcheck/issues/900

Reply:
Right, good catch. I kinda assumed `shellcheck` would catch this but :shrug: